docs/docs/concepts/environment-tokens.mdx

```diff
@@ -24,4 +24,4 @@ Create a new token by clicking on the `Create a new token` button, choosing a na
 tracetest configure --token {ADD_YOUR_TOKEN_HERE}
 ```
 
-One example of this usage can be seen [here](https://github.com/kubeshop/tracetest/blob/main/.github/workflows/scheduled-jobs.yml#L23).
```

Comment:
Should we reference the line that configures the token or the whole file?

https://github.com/kubeshop/tracetest/blob/main/.github/workflows/synthetic-monitoring-production.yml#L24